[2024-09-21T20:17:11.534+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_ingestion.spark_raw_ingestion_empresa scheduled__2024-09-20T15:00:00+00:00 [queued]>
[2024-09-21T20:17:11.552+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_ingestion.spark_raw_ingestion_empresa scheduled__2024-09-20T15:00:00+00:00 [queued]>
[2024-09-21T20:17:11.553+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 0
[2024-09-21T20:17:11.579+0000] {taskinstance.py:1382} INFO - Executing <Task(SSHOperator): spark_raw_ingestion_empresa> on 2024-09-20 15:00:00+00:00
[2024-09-21T20:17:11.585+0000] {standard_task_runner.py:57} INFO - Started process 139 to run task
[2024-09-21T20:17:11.588+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'raw_ingestion', 'spark_raw_ingestion_empresa', 'scheduled__2024-09-20T15:00:00+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/raw_ingestion.py', '--cfg-path', '/tmp/tmp4j_qyjmf']
[2024-09-21T20:17:11.591+0000] {standard_task_runner.py:85} INFO - Job 278: Subtask spark_raw_ingestion_empresa
[2024-09-21T20:17:11.772+0000] {task_command.py:416} INFO - Running <TaskInstance: raw_ingestion.spark_raw_ingestion_empresa scheduled__2024-09-20T15:00:00+00:00 [running]> on host airflow-worker
[2024-09-21T20:17:12.337+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_ingestion' AIRFLOW_CTX_TASK_ID='spark_raw_ingestion_empresa' AIRFLOW_CTX_EXECUTION_DATE='2024-09-20T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-20T15:00:00+00:00'
[2024-09-21T20:17:12.339+0000] {ssh.py:141} INFO - Creating ssh_client
[2024-09-21T20:17:12.340+0000] {ssh.py:114} INFO - ssh_hook is not provided or invalid. Trying ssh_conn_id to create SSHHook.
[2024-09-21T20:17:12.416+0000] {base.py:73} INFO - Using connection ID 'spark_ssh_default' for task execution.
[2024-09-21T20:17:12.417+0000] {ssh.py:300} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-09-21T20:17:12.435+0000] {transport.py:1893} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-09-21T20:17:12.615+0000] {transport.py:1893} INFO - Authentication (password) successful!
[2024-09-21T20:17:12.617+0000] {ssh.py:478} INFO - Running command: export SPARK_HOME=/usr/local/spark; /opt/conda/bin/spark-submit --master local[*] --executor-memory 2g /opt/shared/scripts/raw_ingestion.py 2024-09-20 empresa
[2024-09-21T20:17:22.017+0000] {ssh.py:524} INFO - [DEBUG] dt_now: 2024-09-20
[2024-09-21T20:17:22.018+0000] {ssh.py:524} INFO - [DEBUG] endpoint: empresa
[2024-09-21T20:17:22.575+0000] {ssh.py:524} INFO - 24/09/21 20:17:22 INFO SparkContext: Running Spark version 3.3.1
[2024-09-21T20:17:22.635+0000] {ssh.py:524} INFO - 24/09/21 20:17:22 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-09-21T20:17:22.966+0000] {ssh.py:524} INFO - 24/09/21 20:17:22 INFO ResourceUtils: ==============================================================
[2024-09-21T20:17:22.966+0000] {ssh.py:524} INFO - 24/09/21 20:17:22 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-09-21T20:17:22.967+0000] {ssh.py:524} INFO - 24/09/21 20:17:22 INFO ResourceUtils: ==============================================================
[2024-09-21T20:17:22.967+0000] {ssh.py:524} INFO - 24/09/21 20:17:22 INFO SparkContext: Submitted application: raw_ingestion_empresa
[2024-09-21T20:17:23.047+0000] {ssh.py:524} INFO - 24/09/21 20:17:23 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-09-21T20:17:23.073+0000] {ssh.py:524} INFO - 24/09/21 20:17:23 INFO ResourceProfile: Limiting resource is cpu
[2024-09-21T20:17:23.079+0000] {ssh.py:524} INFO - 24/09/21 20:17:23 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-09-21T20:17:23.435+0000] {ssh.py:524} INFO - 24/09/21 20:17:23 INFO SecurityManager: Changing view acls to: root
[2024-09-21T20:17:23.436+0000] {ssh.py:524} INFO - 24/09/21 20:17:23 INFO SecurityManager: Changing modify acls to: root
[2024-09-21T20:17:23.448+0000] {ssh.py:524} INFO - 24/09/21 20:17:23 INFO SecurityManager: Changing view acls groups to: 
[2024-09-21T20:17:23.452+0000] {ssh.py:524} INFO - 24/09/21 20:17:23 INFO SecurityManager: Changing modify acls groups to: 
[2024-09-21T20:17:23.457+0000] {ssh.py:524} INFO - 24/09/21 20:17:23 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(root); groups with view permissions: Set(); users  with modify permissions: Set(root); groups with modify permissions: Set()
[2024-09-21T20:17:26.271+0000] {ssh.py:524} INFO - 24/09/21 20:17:26 INFO Utils: Successfully started service 'sparkDriver' on port 45719.
[2024-09-21T20:17:28.457+0000] {ssh.py:524} INFO - 24/09/21 20:17:28 INFO SparkEnv: Registering MapOutputTracker
[2024-09-21T20:17:29.419+0000] {ssh.py:524} INFO - 24/09/21 20:17:29 INFO SparkEnv: Registering BlockManagerMaster
[2024-09-21T20:17:29.538+0000] {ssh.py:524} INFO - 24/09/21 20:17:29 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-09-21T20:17:29.539+0000] {ssh.py:524} INFO - 24/09/21 20:17:29 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-09-21T20:17:29.813+0000] {ssh.py:524} INFO - 24/09/21 20:17:29 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-09-21T20:17:30.143+0000] {ssh.py:524} INFO - 24/09/21 20:17:30 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-4078c8ec-62df-4060-b1b2-23af03ca4274
[2024-09-21T20:17:30.379+0000] {ssh.py:524} INFO - 24/09/21 20:17:30 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-09-21T20:17:30.607+0000] {ssh.py:524} INFO - 24/09/21 20:17:30 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-09-21T20:17:34.349+0000] {ssh.py:524} INFO - 24/09/21 20:17:34 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2024-09-21T20:17:34.353+0000] {ssh.py:524} INFO - 24/09/21 20:17:34 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2024-09-21T20:17:34.355+0000] {ssh.py:524} INFO - 24/09/21 20:17:34 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2024-09-21T20:17:34.360+0000] {ssh.py:524} INFO - 24/09/21 20:17:34 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2024-09-21T20:17:34.365+0000] {ssh.py:524} INFO - 24/09/21 20:17:34 WARN Utils: Service 'SparkUI' could not bind on port 4044. Attempting port 4045.
[2024-09-21T20:17:34.372+0000] {ssh.py:524} INFO - 24/09/21 20:17:34 WARN Utils: Service 'SparkUI' could not bind on port 4045. Attempting port 4046.
[2024-09-21T20:17:34.374+0000] {ssh.py:524} INFO - 24/09/21 20:17:34 WARN Utils: Service 'SparkUI' could not bind on port 4046. Attempting port 4047.
[2024-09-21T20:17:34.376+0000] {ssh.py:524} INFO - 24/09/21 20:17:34 WARN Utils: Service 'SparkUI' could not bind on port 4047. Attempting port 4048.
[2024-09-21T20:17:34.380+0000] {ssh.py:524} INFO - 24/09/21 20:17:34 WARN Utils: Service 'SparkUI' could not bind on port 4048. Attempting port 4049.
[2024-09-21T20:17:34.383+0000] {ssh.py:524} INFO - 24/09/21 20:17:34 WARN Utils: Service 'SparkUI' could not bind on port 4049. Attempting port 4050.
[2024-09-21T20:17:34.384+0000] {ssh.py:524} INFO - 24/09/21 20:17:34 WARN Utils: Service 'SparkUI' could not bind on port 4050. Attempting port 4051.
[2024-09-21T20:17:34.449+0000] {ssh.py:524} INFO - 24/09/21 20:17:34 INFO Utils: Successfully started service 'SparkUI' on port 4051.
[2024-09-21T20:17:35.447+0000] {ssh.py:524} INFO - 24/09/21 20:17:35 INFO Executor: Starting executor ID driver on host spark-master
[2024-09-21T20:17:35.496+0000] {ssh.py:524} INFO - 24/09/21 20:17:35 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2024-09-21T20:17:35.752+0000] {ssh.py:524} INFO - 24/09/21 20:17:35 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 45869.
[2024-09-21T20:17:35.753+0000] {ssh.py:524} INFO - 24/09/21 20:17:35 INFO NettyBlockTransferService: Server created on spark-master:45869
[2024-09-21T20:17:35.761+0000] {ssh.py:524} INFO - 24/09/21 20:17:35 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-09-21T20:17:35.800+0000] {ssh.py:524} INFO - 24/09/21 20:17:35 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, spark-master, 45869, None)
[2024-09-21T20:17:35.825+0000] {ssh.py:524} INFO - 24/09/21 20:17:35 INFO BlockManagerMasterEndpoint: Registering block manager spark-master:45869 with 434.4 MiB RAM, BlockManagerId(driver, spark-master, 45869, None)
[2024-09-21T20:17:35.829+0000] {ssh.py:524} INFO - 24/09/21 20:17:35 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, spark-master, 45869, None)
[2024-09-21T20:17:35.833+0000] {ssh.py:524} INFO - 24/09/21 20:17:35 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, spark-master, 45869, None)
[2024-09-21T20:17:45.888+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 178, in execute
    result = self.run_ssh_client_command(ssh_client, self.command, context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 163, in run_ssh_client_command
    exit_status, agg_stdout, agg_stderr = self.hook.exec_ssh_client_command(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/hooks/ssh.py", line 549, in exec_ssh_client_command
    raise AirflowException("SSH command timed out")
airflow.exceptions.AirflowException: SSH command timed out
[2024-09-21T20:17:45.892+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=raw_ingestion, task_id=spark_raw_ingestion_empresa, execution_date=20240920T150000, start_date=20240921T201711, end_date=20240921T201745
[2024-09-21T20:17:45.920+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 278 for task spark_raw_ingestion_empresa (SSH command timed out; 139)
[2024-09-21T20:17:45.960+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-21T20:17:46.026+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
